domains/luongbinhduong.json

```
@@ -4,6 +4,7 @@
         "email": "lbd.2005.dev@gmail.com"
     },
     "records": {
+        "TXT": "ec59405b6e9c804b1f565738f3a9b4",
         "CNAME": "luongbinhduong.github.io"
     }
```

you cant combine CNAME with other records unless proxied
```suggestion
    },
    "proxied": true
```